src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,37 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService) injector
+                .getInstance(ManufacturerService.class);
+        manufacturerService.getAll().forEach(System.out::println);
+        Manufacturer manufacturer = new Manufacturer("<SONY>", "Japan");
+        manufacturerService.create(manufacturer);
+        manufacturer = new Manufacturer("DELL", "Usa");
+        manufacturer = manufacturerService.create(manufacturer);
+        System.out.println(manufacturerService.get(manufacturer.getId()).toString());
```

Comment:
```suggestion
        System.out.println(manufacturerService.get(manufacturer.getId()));
```
toString() is called automatically